### PR TITLE
📖 update v0.20/README.md for all scenarios to work with latest builds

### PR DESCRIPTION
**Pull request by pdettori**
## Summary

Update the base README for all scenarios to work with latest builds from Jan 17, 2024

This is still an interim README until the docs work is completed.

### CODE REVIEW

**Comment by MikeSpreitzer** [comment]
Thanks!
Do you want to add those docs to the list in https://github.com/kubestellar/kubestellar/blob/ks-0.20/README.md#quickstart ?

File: docs/content/v0.20/README.md [hunk 1]
```diff
@@ -49,7 +50,7 @@ including OCM Klusterlet for the WECs.
 ## Setup
 
 1. Install [KubeFlex](https://github.com/kubestellar/kubeflex/tree/main#installation). 
-Make sure you have at least version 0.3.3 or higher. To upgrade from an existing installation, 
+Make sure you have at least version v0.4.0 or higher. To upgrade from an existing installation,
```
Comment (effi-ofer) [inline docs/content/v0.20/README.md:53]: Need to update the pre-req listing here: https://github.com/kubestellar/kubestellar/blob/ks-0.20/test/pre-reqs.md

File: docs/content/v0.20/README.md [hunk 2]
```diff
@@ -74,12 +75,12 @@ helm --kube-context imbs1 upgrade --install status-addon -n open-cluster-managem
 
 see [here](https://github.ibm.com/dettori/status-addon) for more details on the add-on.
 
-4. Create a Workload Description Space `wds1` directly in KubeFlex. Similarly to before, `-p kslight` 
+4. Create a Workload Description Space `wds1` in KubeFlex. Similarly to before, `-p kubestellar`
```
Comment (effi-ofer) [inline docs/content/v0.20/README.md:78]: Would it make sense to document how to run a local build of kubestellar rather than use the -p kubestellar param?

Comment (pdettori) [reply docs/content/v0.20/README.md:78]: yes, definitely should be documented, but perhaps it should be on a "developer guide" doc ? This doc (or what we will derive from it) should be focused on end users that want to test a release without having to clone the project, install go and build tools and make builds to run the tests.

File: docs/content/v0.20/README.md (hunk 1 above)
Comment (pdettori) [reply docs/content/v0.20/README.md:54]: done

**Comment by pdettori** [comment]
> Thanks! Do you want to add those docs to the list in https://github.com/kubestellar/kubestellar/blob/ks-0.20/README.md#quickstart ?

Done.

**Comment by MikeSpreitzer** [comment]
How tested is this?

**Comment by pdettori** [comment]
> How tested is this?

@MikeSpreitzer It's all tested with the images tagged 0.20.0-alpha.1, which I have built. If there are adjustments to do after the merge / tagging will do in a separate PR.

**Comment by MikeSpreitzer** [comment]
Ideally we would have testing by someone or something besides the author.

**Comment by MikeSpreitzer** [comment]
I understand the plan to be as follows.
1. merge PR 1576 into ks-0.20
2. do the branch work
3. apply the tag v0.20.0-alpha.1 to main
4. rebuild the container images
5. retest the scenarios to verify that they still work

.. before advertising this. That is OK with me.

**Pull request merged**